src/olympia/editors/tests/test_views.py

```diff
@@ -1812,6 +1822,71 @@ def test_clear_search_uses_correct_queue(self):
         assert pq(r.content)('.clear-queue-search').attr('href') == url
 
 
+class TestQueueSearchUnlistedAllList(TestQueueSearch):
+    listed = False
+
+    def setUp(self):
+        super(TestQueueSearchUnlistedAllList, self).setUp()
+        self.url = reverse('editors.unlisted_queue_all')
+
+    def test_not_searching(self):
+        self.generate_files(['Not Admin Reviewed', 'Admin Reviewed'])
+        r = self.search()
+        # Because we're logged in as senior editor we see admin reviewed too.
+        assert sorted(self.named_addons(r)) == [
+            'Admin Reviewed', 'Not Admin Reviewed']
+
+    def test_search_deleted(self):
+        self.generate_files(['Not Admin Reviewed', 'Deleted'])
+        r = self.search(deleted=1)
+        assert self.named_addons(r) == ['Deleted']
+
+    def test_search_not_deleted(self):
+        self.generate_files(['Not Admin Reviewed', 'Deleted'])
+        r = self.search(deleted=0)
+        assert self.named_addons(r) == ['Not Admin Reviewed']
+
+    def test_search_by_guid(self):
+        name = 'Not Admin Reviewed'
+        addon = self.generate_file(name)['addon']
+        addon.update(guid='guidymcguid.com')
+        r = self.search(text_query='mcguid')
+        assert self.named_addons(r) == ['Not Admin Reviewed']
+
+    # No platform search for Unlisted All list.
+    def test_search_by_platform_mac(self):
+        pass
+
+    def test_search_by_platform_linux(self):
+        pass
+
+    def test_search_by_platform_mac_linux(self):
+        pass
+
+    def test_preserve_multi_platform_files(self):
+        pass
+
+    def test_preserve_single_platform_files(self):
+        pass
+
+    # No addon type search for Unlisted All list.
+    def test_search_by_addon_type(self):
+        pass
+
+    def test_search_by_addon_type_any(self):
+        pass
+
+    def test_search_by_many_addon_types(self):
+        pass
+
+    def test_age_of_submission(self):
+        pass
+
+    # We don't show platform/application icons in Unlisted All List.
+    def test_preserve_multi_apps(self):
+        pass
+
```

Comment:
They fail because those fields don't exist on the form but it was convenient to subclass TestQueueSearch, which is testing the other form that has them.  I'll create a super class with the common tests.